tests/conftest.py

```diff
@@ -97,44 +97,85 @@ def softfork_height(request):
     return request.param
 
 
-block_format_version = "rc4"
+block_format_version = "rc5"
```

Comment:
this variable name is not intuitive to me. this sounds like a leftover from before the plot format was nailed down, a release candidate. Right now it just seems to be a version number in the filename.

to me it seems like this variable should be renamed to *just* `block_version` or maybe even `chain_version` and its value should be just `v1`, `v2` etc.

Reply:
yeah it is just a block version, I'll change that.